matRadGUI.m

```diff
@@ -3670,19 +3693,32 @@ function edit_windowRange_Callback(hObject, eventdata, handles)
 %        str2double(get(hObject,'String')) returns contents of edit_windowRange as a double
 
 selectionIndex = get(handles.popupmenu_chooseColorData,'Value');
+vRange = str2num(get(hObject,'String'));
+% matlab adds a zero in the beginning when text field is changed
+if numel(vRange) == 3
+    vRange = vRange(vRange~=0);
+end
 
```

Comment:
Maybe we should check if vRange(1) < vRange(2) and if not issue a warning and swap the two values?